project/pom.xml

```diff
@@ -223,7 +223,7 @@
     <!-- General dependency versions -->
     <gson.version>2.10.1</gson.version>
     <guava.version>32.0.0-jre</guava.version>
-    <guice.version>5.1.0</guice.version>
+    <guice.version>7.0.0</guice.version>
```

Comment:
```suggestion
    <guice.version>6.0.0</guice.version>
```

Reply:
You are welcome to investigate the failures with 6.0.0 but I don't have the time or the interest:

```
 jclouds-compute: Compilation failure
[ERROR] /home/gaul/work/jclouds/compute/src/main/java/org/jclouds/compute/config/BaseComputeServiceContextModule.java:[9
2,71] no suitable method found for toProvider(java.lang.Class<org.jclouds.compute.config.GetLoginForProviderFromProperti
esAndStoreCredentialsOrReturnNull>)
[ERROR]     method com.google.inject.binder.LinkedBindingBuilder.toProvider(com.google.inject.Provider<? extends org.jcl
ouds.domain.LoginCredentials>) is not applicable
[ERROR]       (argument mismatch; java.lang.Class<org.jclouds.compute.config.GetLoginForProviderFromPropertiesAndStoreCr
edentialsOrReturnNull> cannot be converted to com.google.inject.Provider<? extends org.jclouds.domain.LoginCredentials>)
[ERROR]     method com.google.inject.binder.LinkedBindingBuilder.toProvider(javax.inject.Provider<? extends org.jclouds.
domain.LoginCredentials>) is not applicable
[ERROR]       (argument mismatch; java.lang.Class<org.jclouds.compute.config.GetLoginForProviderFromPropertiesAndStoreCredentialsOrReturnNull> cannot be converted to javax.inject.Provider<? extends org.jclouds.domain.LoginCredentials>)
[ERROR]     method com.google.inject.binder.LinkedBindingBuilder.toProvider(java.lang.Class<? extends javax.inject.Provider<? extends org.jclouds.domain.LoginCredentials>>) is not applicable
[ERROR]       (argument mismatch; java.lang.Class<org.jclouds.compute.config.GetLoginForProviderFromPropertiesAndStoreCredentialsOrReturnNull> cannot be converted to java.lang.Class<? extends javax.inject.Provider<? extends org.jclouds.domain.LoginCredentials>>)
[ERROR]     method com.google.inject.binder.LinkedBindingBuilder.toProvider(com.google.inject.TypeLiteral<? extends javax.inject.Provider<? extends org.jclouds.domain.LoginCredentials>>) is not applicable
[ERROR]       (argument mismatch; java.lang.Class<org.jclouds.compute.config.GetLoginForProviderFromPropertiesAndStoreCredentialsOrReturnNull> cannot be converted to com.google.inject.TypeLiteral<? extends javax.inject.Provider<? extends org.jclouds.domain.LoginCredentials>>)
[ERROR]     method com.google.inject.binder.LinkedBindingBuilder.toProvider(com.google.inject.Key<? extends javax.inject.Provider<? extends org.jclouds.domain.LoginCredentials>>) is not applicable
[ERROR]       (argument mismatch; java.lang.Class<org.jclouds.compute.config.GetLoginForProviderFromPropertiesAndStoreCredentialsOrReturnNull> cannot be converted to com.google.inject.Key<? extends javax.inject.Provider<? extends org.jclouds.domain.LoginCredentials>>)
```

Reply:
> I don't have the time or the interest

Neither do I.